source/_integrations/risco.markdown

```diff
@@ -65,7 +65,28 @@ And in the reverse direction:
 | Arm Away | Arm |
 | Arm Home | Partial Arm |
 
-Currently supported plaforms:
+## Services
+
+### Service `risco.bypass_zone`
+
+This service marks a zone as bypassed, so that the alarm isn't triggered when the zone is triggered.
+Note you can only bypass a zone when the partitions it belongs to are disarmed, and it will take effect next time you arm.
+Risco automatically un-bypasses the zone after the alarm is disarmed again.
```

Comment:
Adding some additional space for readability (note that a line break doesn't show in Markdown).

```suggestion
This service marks a zone as bypassed so that the alarm isn't triggered when the zone is triggered.

Note you can only bypass a zone when the partitions it belongs to are disarmed, and it will take effect next time you arm.

Risco automatically un-bypasses the zone after the alarm is disarmed again.
```